loops-and-arrays.js

```diff
@@ -22,6 +25,12 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+for (let i=0; i < numbers.length; i++){
+    if (numbers[i] % 2 === 0) numbers[i] = numbers[i] * 2
+    else { (numbers[i] * 3); numbers[i] = numbers[i] * 3}
+} console.log(numbers)
+
```

Comment:
Sommige gedeeltes van de code zijn redundant, maar zo lang het werkt is het goed.
Wat ik je wel graag zou willen meegeven is dat je wat meer let op de schrijfwijze. 
Zo zijn er niet genoeg enters, en ontbreken er haakjes. Let dus goed op de syntax.